### ISSUE TITLE
Register a User

### ISSUE THREAD

**Issue by cseward1** [cseward1/NSSBackEnd-Project#2]
**Once the user lands on the Register Page...**

the user will see a form
-create a form
-this form will require: First Name, last name, Email, password
**Stretch Goal:** Ask them what genre, authors they like and play with that information when they are redirected to the home page.

Make sure all forms that are required are validated
store that information with yourSQL connection when the user clicks an affordance to **submit** the form. 

If the form was filled out correctly and submitted, the user will be redirected to the home page and allowed to start personalized there account and find friends.
